script/proxy.test.js

```diff
@@ -570,4 +572,131 @@ describe("Test FlowColdStorageProxy contract", () => {
     expect(error).not.toBe(null);
     expect(tx).toBe(null);
   });
+
+  test("Get public key before contract update - ProxyDeployAccount", async () => {
+    const keyIndex = 0;
+    const ProxyDeployAccount = await getAccountAddress("ProxyDeployAccount");
+    const args = [ProxyDeployAccount, keyIndex];
+    const [key, err] = await executeScript({
+      name: "get-public-key",
+      args
+    });
+
+    expect(err).toBe(null);
+    expect(key).not.toBe(null);
+    prePublicKey = key;
+  });
+
+  test("Get key index before contract update - ProxyDeployAccount", async () => {
+    const keyIndex = 0;
+    const ProxyDeployAccount = await getAccountAddress("ProxyDeployAccount");
+    const args = [ProxyDeployAccount, keyIndex];
+    const [key, err] = await executeScript({
+      name: "get-key-index",
+      args
+    });
+
+    expect(err).toBe(null);
+    expect(key).toBe(keyIndex);
+  });
+
+  test("Get revoke status of key before contract update - ProxyDeployAccount", async () => {
+    const keyIndex = 0;
+    const ProxyDeployAccount = await getAccountAddress("ProxyDeployAccount");
+    const args = [ProxyDeployAccount, keyIndex];
+    const [status, err] = await executeScript({
+      name: "get-revoke-status",
+      args
+    });
+
+    expect(err).toBe(null);
+    expect(status).toBe(false);
+  });
+
+  test("Update proxy contract - to ProxyDeployAccount", async () => {
```

Comment:
Could we add a test case with incorrect hash of new key?

Reply:
Yes, I will write the test.